hash/fmtowns_cd.xml

```diff
@@ -3799,6 +4013,297 @@ User/save disks that can be created from the game itself are not included.
+				<rom name="the queen of duellist (user disk).hdm" size="1261568" crc="4be69833" sha1="b41753c9994b71a4c6634807fe437b465defde5a" offset="000000" />
+			</dataarea>
+		</part>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="the queen of duellist" sha1="311993a47783fdad8e1fdc4ed65d388dbf7f7d2d" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="toutrun">
+		<description>Turbo Out Run</description>
+		<year>1989</year>
+		<publisher>CRI</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="turbo outrun" sha1="35a8ec4cf334ee803abfdadd798e1fccee644776" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="townsmg1">
+		<description>Towns Magazine Vol. 1</description>
+		<year>1994</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="towns magazine vol. 1" sha1="eab53db6a65d7d7030da1b9e7c5a93a19bd2f279" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="townsmg2">
+		<description>Towns Magazine Vol. 2</description>
+		<year>1994</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom1" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="towns magazine vol. 2 (disc 1)" sha1="65d208a41d0089eb11f7607e0a78fba60fbdbaa4" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="towns magazine vol. 2 (disc 2)" sha1="241a9f4717ed95fa5c6689665c3330f796a28561" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="trigger">
+		<description>Trigger</description>
+		<year>1994</year>
+		<publisher>ZyX</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="trigger" sha1="4e6040576d084861e6fd478a8fbfb6b6723c7596" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="tss1110">
+		<description>Towns System Software v1.1 L10</description>
+		<year>1989</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="[os] towns system software v1.1 l10" sha1="4cc560ebcf22337d4b6ae7143dafc2410c07907a" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="tss1120">
+		<description>Towns System Software v1.1 L20</description>
+		<year>1989</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="[os] towns system software v1.1 l20" sha1="d866918df900ff885e50da7ab96354cf4b9aae24" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="tss1130">
+		<description>Towns System Software v1.1 L30</description>
+		<year>1990</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="[os] towns system software v1.1 l20" sha1="b67bc80774182e641f713920712410be4e715171" />
```

Comment:
Please rename this one to "[os] towns system software v1.1 l30" to fix the naming conflict with tss1120. Thanks!